core/assumes/sat_checker_test.go

```diff
@@ -98,22 +89,17 @@ assumes:
 `)
 
 	expErr := `
-Charm feature requirements cannot be met:
-  - charm requires feature "storage" (version >= 42.0.0) but model currently supports version 2.19.42
+Charm cannot be deployed because:
+  - charm requires feature "storage" (version >= 42.0.0)
   - charm requires all of the following:
-    - charm requires feature "juju" (version >= 3.0.0) but model currently supports version 2.19.42
```

Comment:
question: why remove "but the model currently supports.."? it looks useful.

Reply:
Yeah you're right, I could put this back in